src/handlers/payout/post.ts

```diff
@@ -0,0 +1,60 @@
+import { getWalletAddress } from "../../adapters/supabase";
+import { getBotConfig, getBotContext, getLogger } from "../../bindings";
+import { addCommentToIssue, generatePermit2Signature, getAllIssueComments, getTokenSymbol, parseComments } from "../../helpers";
+import { Payload } from "../../types";
+
+/**
+ * Incentivize the contributors based on their contribution.
+ * The default formula has been defined in https://github.com/ubiquity/ubiquibot/issues/272
+ */
+export const incentivizeComments = async () => {
+  const logger = getLogger();
+  const {
+    mode: { incentiveMode },
+    price: { baseMultiplier },
+    payout: { paymentToken, rpc },
+  } = getBotConfig();
+  if (!incentiveMode) {
+    logger.info(`No incentive mode. skipping to process`);
+    return;
+  }
+  const context = getBotContext();
+  const payload = context.payload as Payload;
+  const issue = payload.issue;
+  if (!issue) {
+    logger.info(`Not an gh issue, skipping`);
+    return;
+  }
+  const issueComments = await getAllIssueComments(payload.issue?.number!);
+  const issueCommentsByUser: Record<string, string[]> = {};
+  for (const issueComment of issueComments) {
+    const user = issueComment.user;
+    issueCommentsByUser[user.login].push(issueComment.body);
+  }
+  const tokenSymbol = await getTokenSymbol(paymentToken, rpc);
+
+  // The mapping between gh handle and comment with a permit url
+  let reward: Record<string, string> = {};
+
+  // The mapping between gh handle and amount in ETH
+  let fallbackReward: Record<string, string> = {};
+  let comment: string = "";
+  for (const user of Object.keys(issueCommentsByUser)) {
```

Comment:
Bounty hunter and bot should not receive payments. Is it implemented?

Reply:
this was a nice catch! pushing fixes